pkg/integrations/windows_exporter/config.go

```diff
@@ -0,0 +1,40 @@
+// +build windows
```

Comment:
This doesn't need to be marked as windows-only since there's nothing Windows specific in here.